app/preact/src/client/preview/index.ts

```diff
@@ -0,0 +1,29 @@
+import { start } from '@storybook/core/client';
+import { ClientApi } from './types';
+
+import './globals';
+import render from './render';
+
+const api = start(render);
+const { load: coreLoad, clientApi, configApi } = api;
+const framework = 'preact';
+
+export const storiesOf: ClientApi['storiesOf'] = (kind, m) => {
+  return (clientApi.storiesOf(kind, m) as ReturnType<ClientApi['storiesOf']>).addParameters({
+    framework,
+  });
+};
+export const load: ClientApi['load'] = (...args: any) => coreLoad(...args, framework);
+export const {
```

Comment:
@ndelangen I encountered auto fixing of `use object destructuring. prefer-destructuring`. I first thought of just copy/paste from `react, angular`, but tried this style. If the intent was to export things with types so that user can have a typescript advantage, then I tested that this one also works! Just let me know, I can change the code to keep the current style.

Reply:
Yea, I kinda disagreed with eslint on this, I found this a bit harder to understand personally.

Not a deal-breaker, but I prefer the syntax I used in app/react.

Reply:
If I migrate the framework from `app` next time, I'll follow the syntax from `react, angular`